common/persistence/tests/cassandra_test_util.go

```diff
@@ -124,3 +118,56 @@ func TearDownCassandraKeyspace(cfg *config.Cassandra) {
 		panic(fmt.Sprintf("unable to drop Cassandra keyspace: %v", err))
 	}
 }
+
+func GetVersionedSchemaFilesInOrder(schemaDir string, logger log.Logger) []string {
```

Comment:
I found this name really hard to parse.  Since this is the only getter for schema files, unless you anticipate adding more I'd vote for just GetSchemaFiles() (and assume the caller can read a doc string) or perhaps GetSchemaFilesSorted() if you prefer.

Reply:
Done, and added docstring.